app/routes/content.js

```diff
@@ -104,7 +104,8 @@ export default Ember.Route.extend(Analytics, ResetScrollMixin, SetupSubmitContro
                 ['og:description', node.get('description')],
                 ['og:site_name', this.get('theme.provider.name')],
                 ['og:type', 'article'],
-                ['article:published_time', new Date(preprint.get('dateCreated') || null).toISOString()]
+                ['article:published_time', new Date(preprint.get('datePublished')).toISOString()],
```

Comment:
Yes- or identify why it's there....?

Sometimes complex code points to an edge case, so I try to ask before just calling for its removal.... but in this case, the most likely edge case would cause the published date to show "1970". 

If we can simplify the code and get the same result, prefer the simpler version.